### ISSUE TITLE
Set zoom step to 1

### ISSUE THREAD

**Issue by netlen-code** [YACReader/yacreader#287]
For hotkey +, - or slider zoom step must be 1 or with at least adjustable with key combination Ctrl +, Ctrl -.
It is also be good to implement input area with float zoom value.

Any rescale algorithm will produce artifacts on incorrectly target scale of dot filled area. Ex: in manga.
With zoom you can adjust it until artifacts disappear.
Zoom step 5 make this impossible in reasonable sense of zoom value.

Using scenario: double page mode, fit to screen.

**Comment by subokita** [comment]
Yes, zoom increment of 10 is just way too much for some cases. In some monitor, it could mean that the page cannot fit onto the screen, or the font could be way too small to be readable.

Allowing the zoom step of 1, will help a lot.
